contracts/PrizeStrategyUpkeep.sol

```diff
@@ -10,17 +10,19 @@ import "./interfaces/PrizePoolRegistryInterface.sol";
 import "./interfaces/PrizePoolInterface.sol";
 import "./utils/SafeAwardable.sol";
 
+import "@pooltogether/pooltogether-generic-registry/contracts/AddressRegistry.sol";
+
 
 ///@notice Contract implements Chainlink's Upkeep system interface, automating the upkeep of PrizePools in the associated registry. 
 contract PrizeStrategyUpkeep is KeeperCompatibleInterface {
 
     using SafeAwardable for address;
 
-    address public prizePoolRegistry;
+    AddressRegistry public prizePoolRegistry;
 
     uint public upkeepBatchSize;
     
-    constructor(address _prizePoolRegistry, uint256 _upkeepBatchSize) public {
+    constructor(AddressRegistry _prizePoolRegistry, uint256 _upkeepBatchSize) public {
```

Comment:
After running the contract on Rinkeby, we quickly hit 1m gas usage for 5 pools.  Let's make `upkeepBatchSize` configurable by the owner.

Reply:
Done had to make the contract Ownable)